napari/view_layers.py

```diff
@@ -20,15 +20,16 @@ def view_<layer_type>(*args, **kwargs):
 """
 import inspect
 import sys
-from typing import Callable
+import typing
+
 
 from numpydoc.docscrape import NumpyDocString
 
 from .components.add_layers_mixin import AddLayersMixin
 from .viewer import Viewer
 
 
-def _build_view_function(layer_string: str) -> Callable:
+def _build_view_function(layer_string: str, method_name: str = None):
```

Comment:
Rename to `_generate_view_function` or similar now, so that it's clear that this is doing more than just building?